### DISCUSSION TITLE
Necessary Changes in configuration.h

### DISCUSSION THREAD

**Discussion by Schnabulation** [Jyers/Marlin#41] [Q&A] [answered]
Hello all...

I have compared the configuration.h and configuration_adv.h from the branches "Ender_3_V2_Rewrite" and "Ender_3_V2_Extensible_UI" and have found quite a bunch of changes, which of these are needed to be applied in order for the new branch to work?

configuration.h
`#define CREALITY_DWIN_EXTUI`
`#define EXTENSIBLE_UI`
`#define EXTUI_LOCAL_BEEPER`
`//#define DWIN_CREALITY_LCD`

configuration_adv.h
`#define CONFIG_EXAMPLES_DIR "Creality/Ender-3 V2" (removed entirely)`
`#define SHOW_REMAINING_TIME`
`#define USE_M73_REMAINING_TIME`
`#define LONG_FILENAME_HOST_SUPPORT`
`#define SCROLL_LONG_FILENAMES`

I'm especially interested in the changes in configuration.h...

**Comment by Jyers** [comment, marked as answer]
Everything here is is require except for the long filename host support which I just added because its a nice to have. The configuration.h changes are specifically important as they enable the ExtUI display code as apposed to the old dwin code. I will add a picture of those settings to the wiki on flashing firmware.
